tests/jest/BookReader/utils/SelectionObserver.test.js

```diff
@@ -0,0 +1,45 @@
+// @ts-check
+import sinon from "sinon";
+import { SelectionObserver } from "@/src/BookReader/utils/SelectionObserver";
+
+afterEach(() => {
+  sinon.restore();
+});
+
+describe("SelectionStartedObserver", () => {
+  test("_onSelectionChange", () => {
+    const handler = sinon.spy();
+    const observer = new SelectionObserver(".text-layer", handler);
+    const ev = new Event("selectstart", {});
```

Comment:
does this also fire selectionChange event?

Reply:
Whoops! This is outdated, will update.